database/insertData/03_template_E_drugRegistration.js

```diff
@@ -474,14 +848,13 @@ exports.queries = [
                 }
               }
               {
-                id: 1009
                 code: "S3"
                 title: "Product images"
                 index: 2
                 templateElementsUsingId: {
                   create: [
                     {
-                      code: "S3Q1"
+                      code: "S3Q1UploadSamples"
```

Comment:
This question missing `fileCountLimit` even though you've mentioned it in description. ;)